linode/networkingip/framework_resource.go

```diff
@@ -0,0 +1,173 @@
+package networkingip
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/linode/linodego"
+	"github.com/linode/terraform-provider-linode/v2/linode/helper"
+)
+
+func NewResource() resource.Resource {
```

Comment:
Could pages in the `docs` directory be added for the new `linode_networking_ip`, `data.linode_networking_ips`, and `linode_network_ip_assignment` resources?